cumulusci/core/runtime.py

```diff
@@ -18,16 +18,24 @@ def __init__(self, *args, **kwargs):
             "load_project_config", True
         )  # this & below can be added to fn signature in py3
         load_keychain = kwargs.pop("load_keychain", True)
+        allow_global_keychain = kwargs.pop("allow_global_keychain", False)
 
         self.global_config = None
         self.project_config = None
         self.keychain = None
+        self.global_keychain = False
```

Comment:
Consider naming this `is_global_keychain` or `keychain_is_global` so it's more obviously meant to hold a boolean rather than a keychain object.